src/java.net.http/share/classes/jdk/internal/net/http/hpack/ISO_8859_1.java

```diff
@@ -40,9 +40,10 @@
 //
 // The encoding is simple and well known: 1 byte <-> 1 char
 //
-final class ISO_8859_1 {
```

Comment:
OK - this class is actually used in QPack as well as HPack. Same for the `QuickHuffman` class below (notice that this is Quick - not Quic ;-) )

It might be interesting to investigate if we could pull out these classes to a comment location to break the tie between QPack and HPack. Something to think about agter integration maybe.

Reply:
@AlekseiEfimov I am resolving this conversation now. Something we could think about later!